provider/resources.go

```diff
@@ -65,7 +65,7 @@ func refProviderLicense(license tfbridge.TFProviderLicense) *tfbridge.TFProvider
 }
 
 func Provider() tfbridge.ProviderInfo {
-	p := shimv1.NewProvider(cloudamqp.Provider())
+	p := shimv1.NewProvider(cloudamqp.Provider(""))
```

Comment:
Also not sure if this is the right value to pass to the provider constructor.

Reply:
This is fine - this is just to satisfy the constructor for the upstream Provider